ot/lp/cvx.py

```diff
@@ -10,6 +10,7 @@
 import numpy as np
 import scipy as sp
 import scipy.sparse as sps
+import ot
```

Comment:
you shouldn't import pot inside a module.

something with relative path like
```python
from .__init__ import emd
```
is far better since it imports the emd function from ```the __init__.py```